expyfun/_experiment_controller.py

```diff
@@ -843,7 +852,8 @@ def flip(self, when=None):
         # this waits until everything is called, including last draw
         gl.glClear(gl.GL_COLOR_BUFFER_BIT)
         gl.glBegin(gl.GL_POINTS)
-        gl.glColor4f(0, 0, 0, 0)
+        if not self._enable_video:
+            gl.glColor4f(0, 0, 0, 0)
```

Comment:
I noticed that changing the value of `_enable_video` has no effect once `ExperimentController` has been instantiated (presumably because `ec.flip()` will only get evaluated once, at init time, at which point the answer to `if not self._enable_video` is baked in).  I haven't fully tested the ramifications of omitting the `gl.glColor4f(0, 0, 0, 0)` for non-video stimuli, but I wonder if it's worth omitting the `enable_video` param and having separate `flip()` and `flip_video()` methods?  Seems to violate DRY, but might allow more flexible experiment designs.  YAGNI?